components/dashboard/src/teams/Members.tsx

```diff
@@ -188,7 +211,16 @@ export default function() {
                     </Item>)}
             </ItemsList>
         </div>
-        {genericInvite && showInviteModal && <Modal visible={true} onClose={() => setShowInviteModal(false)}>
+        {genericInvite && showInviteModal && <Modal visible={true} onEnter={() => {
```

Comment:
Note: `onEnter` is only triggered if you already have the modal open, and you hit the "Enter" key (e.g. if you do a lot of keyboard navigation). So this is likely not the correct event handler to track this modal. Instead, you should probably track "clicks" on the "Copy Invite URL" button as @gtsiolis suggested. 🙂

Reply:
@jankeromnes thanks for the explanation here! I was thinking about tracking the Copy Invite URL button too, but this would for situations in which users copy the URL manually and invite a new member to the team without us having visibility of  the user sending an invite (or, more specifically, demanding an invite URL) on the analytics end as the event is only caused when the button is pressed.